logs_http.module

```diff
@@ -152,6 +146,27 @@ function logs_http_register_event(array $log_entry) {
   // Remove empty values, to prevent errors in the indexing of the JSON.
   $event = logs_http_array_remove_empty($event);
 
+  if (variable_get('logs_http_debugging_mode', FALSE)) {
+    if (!isset($events['debugging'])) {
```

Comment:
Well, a hook could be added to allow other modules to add even more detail if they needed to. But the module itself should already do what it says if we add that option, I think. So, adding GET, POST and COOKIE information should be done by this module and after that a hook could allow others to alter that result.